src/common/isolatable_hardwares.cpp

```diff
@@ -742,7 +737,7 @@ std::optional<sdbusplus::message::object_path>
             inventoryPathList->begin(), inventoryPathList->end(),
             [&fruInstId, &fruInvPathLookupFunc, this](const auto& path) {
             return fruInvPathLookupFunc(this->_bus, path, fruInstId);
-        });
+            });
```

Comment:
Adjust white space issue

Reply:
It got formatted by itself when I ran clang-format.

Reply:
Then run clang in a separate commit. Also why clang would change something which you didn't change?